src/main/java/parser/TransactionParserImpl.java

```diff
@@ -0,0 +1,43 @@
+package parser;
+
+import fruittransaction.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransactionParserImpl implements TransactionParser {
+    private static final int INDEX_FOR_OPERATION = 0;
+    private static final int INDEX_FOR_FRUITS = 1;
+    private static final int INDEX_FOR_QUANTITY = 2;
+
+    @Override
+    public List<FruitTransaction> parse(List<String> lines) {
+        List<FruitTransaction> completeParseList = new ArrayList<>();
+        for (String line : lines) {
+            String[] splitLineFromLines = line.split(",");
+            FruitTransaction fruitsObjectForCompleteList = new FruitTransaction();
+
+            if (splitLineFromLines[INDEX_FOR_OPERATION]
+                    .equals(FruitTransaction.Operation.BALANCE.getCode())) {
+                fruitsObjectForCompleteList
+                        .setOperation(FruitTransaction.Operation.BALANCE);
+            } else if (splitLineFromLines[INDEX_FOR_OPERATION]
+                    .equals(FruitTransaction.Operation.SUPPLY.getCode())) {
+                fruitsObjectForCompleteList
+                        .setOperation(FruitTransaction.Operation.SUPPLY);
+            } else if (splitLineFromLines[INDEX_FOR_OPERATION]
+                    .equals(FruitTransaction.Operation.PURCHASE.getCode())) {
+                fruitsObjectForCompleteList
+                        .setOperation(FruitTransaction.Operation.PURCHASE);
+            } else {
+                fruitsObjectForCompleteList
+                        .setOperation(FruitTransaction.Operation.RETURN);
+            }
```

Comment:
this solution isn't flexible and every time a new transaction is added, you should add a new 'else if' here. try to write method in Operation enum to get enum value by String